modules/sndfile/sndfile.c

```diff
@@ -79,7 +81,9 @@ static int get_format(enum aufmt fmt)
 }
 
 
-static SNDFILE *openfile(const struct aufilt_prm *prm, bool enc)
+static SNDFILE *openfile(const struct aufilt_prm *prm,
+			 struct stream *strm,
+			 bool enc)
```

Comment:
Just a minor comment: You may put stream in first line. We have 79 chars per line.
```c
static SNDFILE *openfile(const struct aufilt_prm *prm, struct stream *strm,
			 bool enc)
```